spec/features/repair_request_spec.rb

```diff
@@ -298,15 +298,17 @@ def stub_property_temp_annex
       expect(page).to have_css(".hackney-property-warning-label-turquoise")
       click_on 'Raise a repair on this dwelling'
 
-      expect(page).to have_content "CC"
+      expect(page).to have_css('.hackney-cautionary-contact-table')
       expect(page).to have_link("Launch Keyfax", href: "https://www.keyfax.com")
+      expect(find_by_id('hackney_repair_request_description')).to have_content 'CC'
 
       stub_keyfax_get_results_response
 
       visit new_property_repair_request_path('00000666', status: "1", guid: '123456789')
 
-      select "N - Normal"
-      fill_in "Problem description", with: "It's broken"
+      expect(page).to have_content('N - Normal')
```

Comment:
Did you check if this fails if "N - Normal" is not selected?

Reply:
It does not fail... lol